pkg/gameservers/controller_test.go

```diff
@@ -2275,15 +2277,32 @@ func newSingleContainerSpec() agonesv1.GameServerSpec {
 	}
 }
 
-func newPassthroughPortSingleContainerSpec() corev1.PodSpec {
-	return corev1.PodSpec{
-		Containers: []corev1.Container{
-			{Name: "agones-gameserver-sidecar",
-				Image: "container/image",
-				Env:   []corev1.EnvVar{{Name: passthroughPortEnvVar, Value: "TRUE"}}},
-			{Name: "simple-game-server",
-				Image: "container2/image",
-				Ports: []corev1.ContainerPort{{HostPort: 7777, ContainerPort: 555}},
-				Env:   []corev1.EnvVar{{Name: passthroughPortEnvVar, Value: "TRUE"}}}},
+// Assume ports 0 and 2 are Passthrough ports for "example-server" container
+// The annotation would look like autopilot.gke.io/passthrough-port-assignment: '{"example-server":["0","2"]}'
```

Comment:
I think this comment is out of date (it's `{"example-server":[0,2]}`, right?)

Reply:
I think it could still be valid case unless I'm missing something (?) In this method i'm building the PodSpec and the `example-server` container has 3 ports so technically container ports 0 and 2 are still valid. I'm aware that the `HostPort` and `ContainerPort` are not the same value in this piece of code but that's intentional since at this point the webhook hasn't been triggered. Nevertheless, I included another container in this PodSpec and updated the comment.